docs/filters.md

```diff
@@ -222,6 +224,18 @@ Matches issues which were created on a given date, or within a given date range.
 
 Matches issues which were updated in the given number of hours. For example, `updated:<24` would match issues updated in the last day. If a number `n` is given, it is implicitly translated to `<n`.
 
+### updatedByOthers
+
+*Expects a number or  number range*
+
+Matches issues which were updated in the given number of hours by users other than the logged in user. For example, `updated:<24` would match issues updated in the last day by other users. If a number `n` is given, it is implicitly translated to `<n`.
+
```

Comment:
The format for number ranges is given at the top. Maybe in future we can add links pointing back up.

I agree about the duplication. Since these come right after `updated`, it should be enough to just point out the differences in behaviour.